plugins/seedwatch.cpp

```diff
@@ -103,6 +103,10 @@ static bool validate_seed_config(color_ostream& out, PersistentDataItem c)
 {
     int seed_id = get_config_val(c, SEED_CONFIG_ID);
     auto plant = binsearch_in_vector(world->raws.plants.all, &df::plant_raw::index, seed_id);
```

Comment:
I realize you didn't change this, but I'm pretty sure it could be reduced to `df::plant_raw::find(seed_id)`

Reply:
that was my doing, i forgot about `find`